sidebars.js

```diff
@@ -41,6 +41,7 @@ const sidebars = {
         'ingestion/open-telemetry',
         'ingestion/elastic-search',
         'ingestion/loki',
+        'ingestion/auto-instrument-java-app'
```

Comment:
do not add here. Create new section `Instrumention for traces` and create subsection `Using Java app`